File: segundo/1er cuatri/PDOO/ejercicios_clase/ejercicios_clase/T1S2.rb
# frozen_string_literal: true
require_relative 'monstruos/monstruo'
require_relative 'monstruos/s'

class T1S2
  Monstruo.add_random_monstruo
  Monstruo.add_random_monstruo
  Monstruo.add_random_monstruo
  Monstruo.add_random_monstruo
  Monstruo.add_random_monstruo
  Monstruo.add_random_monstruo
  Monstruo.add_random_monstruo


  ut = Utils.new
  mons = ut.buscar_monstruo("Monstruo5")
  mons.mostrar_info

end

File: segundo/1er cuatri/PDOO/ejercicios_clase/p1/T1S2.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ejercicios_clase;

import ejercicios_clase.monstruos.Monstruo;
import ejercicios_clase.Utils.Utils;
import ejercicios_clase.monstruos.EnumMonstruos;

/**
 *
 * @author alvarogz
 */
public class T1S2 {
    public static void main(String[] args) {
        
        Monstruo m = new Monstruo(Utils.generarNombreAleatorio(), Utils.generatTipoAleatorio(), Utils.generarVidaAleatoria(), Utils.generarAtaqueAleatorio());
        m.mostrar_info();
        
    }
}
